components/brave_wallet/browser/swap_service_unittest.cc

```diff
@@ -322,4 +371,192 @@ TEST_F(SwapServiceUnitTest, IsSwapSupported) {
   EXPECT_FALSE(IsSwapSupported("invalid chain_id"));
 }
 
+TEST_F(SwapServiceUnitTest, GetJupiterQuoteURL) {
+  auto url = swap_service_->GetJupiterQuoteURL(GetCannedJupiterQuoteParams(),
+                                               mojom::kSolanaMainnet);
+  ASSERT_EQ(url,
+            "https://quote-api.jup.ag/v1/quote?"
+            "inputMint=So11111111111111111111111111111111111111112&"
+            "outputMint=EPjFWdd5AufqSSqeM2qN1xzybapC8G4wEGGkZwyTDt1v&"
+            "amount=10000&"
+            "feeBps=85&"
+            "slippagePercentage=1.000000");
+}
+
+TEST_F(SwapServiceUnitTest, GetJupiterSwapTransactionsURL) {
+  auto url =
+      swap_service_->GetJupiterSwapTransactionsURL(mojom::kSolanaMainnet);
+  ASSERT_EQ(url, "https://quote-api.jup.ag/v1/swap");
+}
+
+TEST_F(SwapServiceUnitTest, GetJupiterQuote) {
```

Comment:
For all uint64 values, we need test case to test them with UINT64_MAX to make sure we are handling them correctly.

Reply:
Also, we should have more error test cases, such as negative or non-integer values.

Reply:
Added extensive unit-tests for uint64 fields here: 12ae1ed82ecea61693deebe22936d3ca29888bf2